docs/Getting Started.md

```diff
@@ -0,0 +1,113 @@
+
+2) Check that Hokusai is correctly configured for your project
+
+  `hokusai check` will run a series of tests and warn you if it finds anything improperly configured.
+
+3) Run the development stack
+
+  `hokusai dev start` will build a Docker image, then start a Docker Compose stack defined by `./hokusai/development.yml`
+
+  If this command throws no errors, try interacting with the running stack with:
+
+  ```bash
+  hokusai dev status
+  hokusai dev logs
+  hokusai dev shell
+  ```
+
+  To shut down the stack's running containers, run `hokusai dev stop`
+
+  Container filesystems will be preserved between stack starts and stops, unless you run:
+
+  `hokusai dev clean`, in which case the container filesystems will be deleted.
+
+
+4) Run the test suite in the test stack
+
+  `hokusai test` will build a Docker image, start a Docker Compose stack defined by `./hokusai/test.yml`, run the defined test command in the main (project-name) container to completion, and return its exit code.
+
+  See [Writing Docker Compose YAML files](./Configuration.md#Writing Docker Compose YAML files) if you want to modify your test stack's configuration.
+
+5) Build and push an image to ECR
+
+  `hokusai push` will build and push an image to ECR.  By default, it tags the image as the SHA1 of the HEAD of your current git branch (by calling `git rev-parse HEAD`).  You can override this behavior with the `--tag` option, although this is not recommended as creating builds matched to the SHA1 of Git tags gives you a clean view of your ECR project repository and deployment history.
+
+  The command will also tag the image as `latest`.  This image tag should not be referenced in any Kubernetes YAML configuration, but serves only as a pointer, which is referenced when creating a Kubernetes stack.
+
+  The command aborts if any of the following conditions is met:
+  - The working directory is not clean (you have uncommit changes)
+  - The working directory contains any files specified in your `.gitignore` file
+  - The ECR project repository already contains the specified tag
+
+  The reason for these conditional checks is that when building, Docker will copy your _entire_ working directory into the container image, which can produce unexpected results when building images locally, destined for production environments!  The reason Hokusai aborts if it detects the working directory is unclean, or any ignored files or directories are present, is because it attempts to prevent any local configuration leaking into container images.
+
+  You can skip these conditional checks by supplying the `--force` flag.  Be careful of using `--force` when you have uncommited changes or ignored files in your working directory, especially when supplying `--force` without also supplying the `--tag` option.
+
+  Once an image is pushed, you can list the images and tags in the ECR project repository with: `hokusai images`.
```

Comment:
I think this section could be confusing to someone who is 'getting started' with hokusai. It is quite complex and filled with notes and caveats. I would probably leave out the `--force` details and aim to slim down the rest of it. 

Perhaps a note that this is only required if you haven't hooked up CI to push an image for you (which should be the recommended flow). Admittedly that is outside the scope of this Hokusai centric guide, but maybe a note would clarify that a dev should probably not be doing this frequently.